app/views/hyrax/collections/show.html.erb

```diff
@@ -35,7 +35,7 @@
           <div class="hyc-bugs">
             <div class="hyc-item-count">
               <b><%= @presenter.total_viewable_items %></b>
-              <%= 'Item'.pluralize(@presenter.total_viewable_items) %></div>
+              <%= pluralize(@presenter.total_viewable_items, t('item_count')) %></div>
```

Comment:
This is failing in CI tests.  Maybe this will fix it?
```suggestion
              <%= pluralize(@presenter.total_viewable_items, t('.item_count')) %></div>
```

Reply:
thanks! just a typo